src/wp-includes/functions.php

```diff
@@ -7637,12 +7637,40 @@ function wp_unique_id( $prefix = '' ) {
 function wp_cache_get_last_changed( $group ) {
 	$last_changed = wp_cache_get( 'last_changed', $group );
 
-	if ( ! $last_changed ) {
-		$last_changed = microtime();
-		wp_cache_set( 'last_changed', $last_changed, $group );
+	if ( $last_changed ) {
+		return $last_changed;
 	}
 
-	return $last_changed;
+	return wp_cache_set_last_changed( $group );
```

Comment:
So for sites without object cache, this will be called multiple sites per page load. Meaning doing another get and calling an action on the FE. Is there a possibility that plugins might hook in here and call an action to some cache invalidation on the FE?

Reply:
That wasn't my question. If I ( a developer of a plugin ) call 
```
add_action( 'wp_cache_set_last_changed', function() {
   // Do some crazy cache invalidation here. 
});
```

If this action is called on every page load, this could get very expensive, compute wise.

Reply:
I don't know what to say. That's WordPress I guess?

We could only fire the action when `$_wp_using_ext_object_cache` is true?

Reply:
Yeah, I think it's the nature of the beast and probably a documentation issue.

The thoughts I had behind this filter:

* persistent caches can use it as a trigger to flush the `x-queries` group, preferably in a nice performant manner
* A memory management plugin may use it to call `WP_Object_Cache::flush_group()` on the `*-queries` group (an `unset()`

Is it worth adding this to the filter's docs?